src/components/navbar.js

```diff
@@ -1,21 +1,38 @@
 import React, { Component } from 'react';
 import NavItem from './nav_item';
+import { GoogleLogin, GoogleLogout } from 'react-google-login';
+import axios from 'axios';
+import env from './environment';
 
-import DALIwhiteLogo from '../../Assets/imgs/DALIwhiteLogo.png';
-
-const pages = ['Overview', 'Projects', 'Members', 'Blog', 'Gallery', 'Leaderboard'];
-const links = ['/', '/projects', '/members', '/', '/'];
+import DALIblackLogo from '../../Assets/imgs/DALIblackLogo.png';
+import DALIblueLogo from '../../Assets/imgs/DALIblueLogo.png';
 
 class NavBar extends Component {
   constructor(props) {
     super(props);
 
+    const token = window.localStorage.token;
```

Comment:
generally preferred to use the functional interface to localStorage.  so window.localStorage.getItem('token') is preferred to be parallel to setItem, removeItem etc.  Some reasons for this is that while the indexing method works it might not work for certain keys - like if you had 'length' as a key for some reason.  another is just to keep it parallel.